apps/docs/src/guide/npm-create-fuels/deploying-a-dapp-to-testnet.md

```diff
@@ -0,0 +1,84 @@
+# Deploying a dApp to Testnet
+
+In this guide, we will deploy a full-stack dApp bootstrapped with `npm create fuels` to the Fuel beta-5 testnet.
+
+> Make sure you have already bootstrapped a dApp using `npm create fuels`. If you haven't, please follow [this guide](./index.md).
+
+There are mainly two steps to get our dApp live on the testnet:
+
+1. Deploying the Contract to the Testnet
+2. Deploying the Frontend to the Cloud
+
+## Deploying the Contract
+
+We recommend using [`forc`](https://docs.fuel.network/docs/forc/) to deploy your contracts to the testnet.
+
+`forc` comes bundled in with the Fuel toolchain. To install the Fuel toolchain, run the following command:
+
+```sh
+curl https://install.fuel.network | sh
+```
+
+Once you have the toolchain installed, `cd` into the directory containing your contract:
+
+```sh
+cd sway-programs/contract
+```
+
+And then, run the following command and follow the instructions to deploy the contract to the testnet:
+
+```sh
+forc deploy --testnet
+```
+
+> You can check out [this guide](https://docs.fuel.network/docs/intro/quickstart-contract/#deploy-to-testnet) for more information on deploying a contract to the testnet.
+
+You should see a message similar to this:
+
+```md
+Contract deploy-to-beta-5 Deployed!
+
+Network: https://beta-5.fuel.network
+Contract ID: 0x8342d413de2a678245d9ee39f020795800c7e6a4ac5ff7daae275f533dc05e08
```

Comment:
This is a suggestion but maybe we could just instruct the user to set this as an env var and then pull from there.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2052/commits/344fc22adbbf7dcdf9aee9d045b500816223c823